mm2src/coins/z_coin.rs

```diff
@@ -999,11 +1010,10 @@ impl<'a> ZCoinBuilder<'a> {
     }
 
     async fn init_blocks_db(&self) -> Result<BlockDbImpl, MmError<ZcoinClientInitError>> {
-        let cache_db_path = self.db_dir_path.join(format!("{}_cache.db", self.ticker));
```

Comment:
Why is this removed?

Reply:
thanks, restored (I missed the non-test version of BlockDbImpl::new which does use cache_db_path)